src/identifierToProjectId.js

```diff
@@ -4,6 +4,10 @@ var _ = require("lodash");
 
 var api = require("./api");
 
+/**
+ * This function is deprecated due to its functionality is no longer relevant for the commands.
+ * TODO: remove this function
```

Comment:
nit: "remove this function when tools:migrate is removed" - that way we know when to remove it!